app/javascript/packs/admin/editUser.jsx

```diff
@@ -1,4 +1,12 @@
+import { h, render } from 'preact';
+import { TagAutocompleteSelection } from '../../article-form/components/TagAutocompleteSelection';
```

Comment:
These imports are currently specific to the article editor. Is there anything article editor specific about these @aitchiss or could they become part of the folder where the multi-autocomplete resides?

Reply:
Yeah I guess we either move it to a shared component, or we move it to the multi-select autocomplete folder. The latter kinda suggests to me that we _always_ want to render tags like this (which we probably do?)

Reply:
We can always move it again or rename it if we change our minds 😄

Reply:
No thoughts here. We can probably move it all to live in one place.